components/eam/bld/configure

```diff
@@ -2562,11 +2566,14 @@ sub write_filepath
           print $fh "$camsrcdir/eam/src/physics/crm/samomp/SGS_TKE\n";
 
           # advection scheme
-          my $crm_adv = $cfg_ref->get('crm_adv');
-          if ($crm_adv eq 'UM5') {
-              print $fh "$camsrcdir/eam/src/physics/crm/samomp/ADV_UM5\n";
+          my $crm_adv = 'MPDATA';
+          if (defined $opts{'crm_adv'}) { my $crm_adv = $cfg_ref->get('crm_adv'); }
```

Comment:
Same as for the previous instance, need to remove "`my`" before $crm_adv within the if-logic

Reply:
done.